src/api/api.utils.ts

```diff
@@ -0,0 +1,3 @@
+import { BASE_API_URL } from './api.const';
+
+export const getBaseUrl = () => sessionStorage.getItem('apiUrl') || BASE_API_URL;
```

Comment:
You cold omit gettina and setting api url from the session storage, because login to the Applet Library does not have select with possibility to change api url. In this case you don't need this function at all, just use BASE_API_URL constant.